### ISSUE TITLE
Install Error - Error opening flogo app project: Invalid project, source directory doesn't exists

### ISSUE THREAD

**Issue by josenspire** [TIBCOSoftware/flogo-contrib#313]
**I'm submitting a ...**  (check one with "x")
```
[] bug report => search github for a similar issue or PR before submitting
[] enhancement request
[] feature request
[x] support request
[] general question
```

**Current behavior (how does the issue manifest):**
Follow up the install guide, run with command: 
```
flogo install github.com/TIBCOSoftware/flogo-contrib/activity/log
```
But finally an error has occurred, 
```
Error opening flogo app project: Invalid project, source directory doesn't exists
```
I had search the question in Issues, there was a issue like this, but it's solution is upgrade the cli version since he is used 0.5.6 before. This can not solved my issue.

**Expected behavior:**

Expected it can install the component what i need.

**Minimal steps to reproduce the problem (not required if feature enhancement):**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment (Operating system, docker version, browser & verison if webui, etc):**

Win 10 * 64, docker 2.0.0, chrome.

**Flogo version (CLI & contrib/lib. If unknown, leave empty or state unknown):** 0.X.X

flogo cli version [v0.5.7]

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by retgits** [comment]
Hi! Just to check, did you run the command “flogo install” after running “flogo create”? The latter command will create the outline of an application for you and the former command will allow you to install new activities into an existing app.

**Comment by mellistibco** [comment]
@josenspire yes, first you need to create the project structure, as @retgits had said. Then you'll `cd` into that dir and run the `flogo install` command. If you create an application using the `-f` switch and specify an existing application that you created via the Web UI, then the CLI will install all the dependencies for you.
